core/src/main/scala/com/pingcap/tispark/utils/TiUtil.scala

```diff
@@ -243,4 +244,20 @@ object TiUtil {
 
     mutableRow
   }
+
+  def extractDAGReq(df: DataFrame): TiDAGRequest = {
+    val executedPlan = df.queryExecution.executedPlan
+    val copRDD = executedPlan.find(e => e.isInstanceOf[ColumnarCoprocessorRDD])
+    val regionTaskExec = executedPlan.find(e => e.isInstanceOf[ColumnarCoprocessorRDD])
```

Comment:
`copRDD` is same as `regionTaskExec` ?

Reply:
change to `val regionTaskExec = executedPlan.find(e => e.isInstanceOf[ColumnarRegionTaskExec])` ?